### PR TITLE
Updating texlive path to current version

### PR DESCRIPTION
**Pull request by magic-lantern**
This minor path change fixes the issue #90 as reported by @flowgunso

### CODE REVIEW

**Review by jubilee2 — APPROVED** [review]
i think it is good for merge

File: Dockerfile-base [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN echo "selected_scheme scheme-basic" >> /install-tl-unx/texlive.profile; \
 RUN rm -r /install-tl-unx; \
 	rm install-tl-unx.tar.gz
 
-ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/texlive/2017/bin/x86_64-linux/
+ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/texlive/2018/bin/x86_64-linux/
```
Comment (prohde) [inline Dockerfile-base:27]: Maybe this should be updated to version 2019 now?

`ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/texlive/2019/bin/x86_64-linux/`

**Review by prohde — CHANGES_REQUESTED** [review]
Maybe this should be updated to TeX Live version 2019 now?

**Comment by mserranom** [comment]
Outdated.

**Pull request closed without merging**